.github/workflows/ci.yml

```
@@ -265,16 +217,10 @@ jobs:
           mkdir -p ${{ runner.temp }}/signaldb-data
           chmod 755 ${{ runner.temp }}/signaldb-data
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)